stock_packaging_usability_ul/README.rst

```diff
@@ -1,24 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+============================
 Stock Packaging Usability UL
 ============================
 
+This module allows to select the logistics unit (e.g. EUR-pallet, box 50x40x30, package 30x20x17, ...) on the fly when you define a new pack.
+
+Configuration
+=============
+
+Go to the Settings page of the Inventory menu and enable the option
```

Comment:
Better enumerated list for procedure (`#. `)

Reply:
Same answer.

Reply:
I am thinking about something similar to:
```
#. Go to `Inventory > Configuration > Settings`  
#. Enable the option *Record packages used on packing: pallets, boxes, ...*
#. All users are added to the group *Manage Packages*.
```
Anyway, nice to have :smile: